### PR TITLE
Add IR page titles

### PR DESCRIPTION
**Pull request by iwootten**
### What is the context of this PR?

Individual response currently has missing page titles, they should be updated as per the spreadsheet in the card.

### How to review 
Use `test_individual_response.json` Ensure the page titles now match those in the spreadsheet.

### Checklist

* [x] New static content marked up for translation

### CODE REVIEW

**Comment by iwootten** [comment]
[Add page titles when requesting an individual response (M)](https://trello.com/c/DwhIFGyw/4182-add-page-titles-when-requesting-an-individual-response-m)

**Review by ajmaddaford — CHANGES_REQUESTED** [review]
We already do person number substitution in for Question and Interstitial pages. I understand that that won't work for this as it relies on the questionnaire schema, but these changes should be consistent. 

- The placeholders should use `{}` rather than `%()s`
- The placeholder in the string should be `{list_item_position}`
- The interpolation should be handled by a common method, like `_set_page_title` does in the block handler

File: app/views/handlers/individual_response.py [hunk 1]
```diff
@@ -73,24 +73,34 @@ def __init__(
...
-    def _is_location_valid(self):
-        self._list_name = self._schema.get_individual_response_list()
+    @cached_property
+    def _list_model(self):
+        return self._questionnaire_store.list_store[self._list_name]
 
-        list_model = self._questionnaire_store.list_store[self._list_name]
+    @cached_property
+    def _list_item_position(self):
+        return self._list_model.index(self._list_item_id)
```
Comment (ajmaddaford) [inline app/views/handlers/individual_response.py:86]: The person number is off by one. Using the list store `list_item_position` method would fix this.

File: app/views/handlers/individual_response.py [hunk 2]
```diff
@@ -73,24 +73,34 @@ def __init__(
...
+
+    def get_full_page_title(self, page_title):
+        if self._list_item_id:
+            page_title += ": Person {list_item_position}".format(
```
Comment (ajmaddaford) [inline app/views/handlers/individual_response.py:90]: The separator (`: `) should be separate from the rest of the string, and `Person {list_item_position}` needs to be wrapped in a `lazy_gettext` call.

Comment (JamesGardiner) [reply app/views/handlers/individual_response.py]: Done in 8e5d7c91

File: app/views/handlers/individual_response.py (hunk 1 above)
Comment (JamesGardiner) [reply app/views/handlers/individual_response.py]: Done in 8e5d7c91

File: app/views/handlers/individual_response.py [hunk 3]
```diff
@@ -73,24 +73,38 @@ def __init__(
...
+    @cached_property
+    def _list_model(self):
+        return self._questionnaire_store.list_store[self._list_name]
 
-        list_model = self._questionnaire_store.list_store[self._list_name]
+    @cached_property
+    def _list_item_position(self):
+        return self._questionnaire_store.list_store.list_item_position(
+            self._list_item_id
+        )
```
Comment (ajmaddaford) [inline app/views/handlers/individual_response.py:88]: `list_item_position` takes two arguments - the list name and the list item id.

Comment (JamesGardiner) [reply app/views/handlers/individual_response.py:88]: yeah just realised this failed - fixed in latest push

**Review by ajmaddaford — APPROVED** [review]

File: app/routes/individual_response.py [hunk 4]
```diff
@@ -250,4 +254,7 @@ def individual_response_text_message_confirmation(schema, questionnaire_store):
     return render_template(
         template="individual_response/confirmation-text-message",
         mobile_number=mobile_number,
+        page_title=individual_response_handler.get_full_page_title(
```
Comment (LJBabbage) [inline app/routes/individual_response.py:257]: Not sure get_full_page_title adds much for me, in the other handlers we do similar, but we don't call it full_page_title

**Review by LJBabbage — CHANGES_REQUESTED** [review]
Should we not have a test for this as well?

File: app/routes/individual_response.py (hunk 4 above)
Comment (JamesGardiner) [reply app/routes/individual_response.py:257]: are you suggesting just calling it `page_title`?

Comment (LJBabbage) [reply app/routes/individual_response.py:257]: yep

Comment (JamesGardiner) [reply app/routes/individual_response.py]: Done in 864439b9

**Review by LJBabbage — APPROVED** [review]
Just needs formatting

**Pull request merged**